do_snowDA.sh

```diff
@@ -231,7 +305,7 @@ fi
 
 if [[ $do_DA == "YES" ]]; then 
 
-cat << EOF > apply_incr_nml
```

Comment:
For the true ensemble case, this will need to called for each ensemble member, as the LETKF calculates an increment for each ensemble member.